nevergrad/optimization/optimizerlib.py

```diff
@@ -860,6 +860,44 @@ def _internal_provide_recommendation(self) -> tp.ArrayLike:
         return self.avg
 
 
+class RescaleOptimizer(base.Optimizer):
+    """Proposes a version of a base optimizer which works at a different scale."""
+    def __init__(
+            self,
+            parametrization: IntOrParameter,
+            budget: tp.Optional[int] = None,
+            num_workers: int = 1,
+            base_optimizer: base.OptCls = CMA,
+            scale: tp.Optional[float] = None,
+    ) -> None:
+        super().__init__(parametrization, budget=budget, num_workers=num_workers)
+        self.base_optimizer = base_optimizer(parametrization, budget=budget, num_workers=num_workers)
+        if scale is None:
+            assert budget is not None, "Either scale or budget must be known in RescaleOptimizer."
+            scale = np.sqrt(np.log(self.budget) / self.dimension)
```

Comment:
is that the only rule that you plan to use? if you want to add others, better do it now, and have scale be a float or a string

Reply:
I have no idea of other rule --- this one is quite principled given the maths in Meunier et al.